swagger.json

```diff
@@ -1118,6 +1118,58 @@
             }
           }
         },
+        "/fleet/drivers/{driver-id}/log_edits": {
+            "get": {
+              "tags": [
+                "Fleet"
+              ],
+              "summary": "/fleet/drivers/{driver-id:[0-9]+}/log_edits",
+              "description": "Get a summary of the carrier edits for a specified driver. Note: This does not include edits made by driver",
+              "operationId": "getLogEdits",
+              "parameters": [
+                {
+                    "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                    "name": "driver-id",
+                    "in": "path",
+                    "required": true,
+                    "description": "ID (integer) of the driver with HOS logs.",
+                    "type": "int64"
```

Comment:
This should be type number, format int64.

Reply:
fixed!